accessibility-checker.php

```diff
@@ -360,7 +369,16 @@ function edac_summary( $post_id ) {
 	$summary['passed_tests'] = round( count( $rules_passed ) / count( $rules ) * 100 );
 
 	// phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.DirectDatabaseQuery.NoCaching -- Using direct query for interacting with custom database, safe variable used for table name, caching not required for one time operation.
-	$summary['errors'] = intval( $wpdb->get_var( $wpdb->prepare( 'SELECT count(*) FROM %i where siteid = %d and postid = %d and ruletype = %s and ignre = %d', $table_name, get_current_blog_id(), $post_id, 'error', 0 ) ) );
+	$summary['errors'] = (int) $wpdb->get_var(
+		$wpdb->prepare(
+			'SELECT count(*) FROM %i where siteid = %d and postid = %d and ruletype = %s and ignre = %d',
+			$table_name,
+			get_current_blog_id(),
+			$post_id,
+			'error',
+			0
+		)
+	);
```

Comment:
Split this one (and others below) to multiple lines, but also removed the `intval()` call, replacing it with an `(int)` typecase which is faster.